Makefile

```diff
@@ -1,7 +1,13 @@
 DEV_IMAGE_ORG = jboss-dataservices
-
+REGISTRY =
```

Comment:
We already use `_REGISTRY_IP` variable which points to the internal OpenShift Docker Registry (https://github.com/jboss-container-images/jboss-dataservices-image/blob/dataservices-dev/Makefile#L20). This one looks very similar to that and has absolutely different meaning. How about naming it as `DOCKER_REGISTRY_PREFIX`?